### DISCUSSION TITLE
Version 3.2.0 not available

### DISCUSSION THREAD

**Discussion by ElHado** [Kozea/Radicale#1491] [External Packaging]
Hi, 

I made a fresh install and I don't have the actual version. I got 3.1.9, but 3.2.0 already release two weeks ago. 

pip say it's the newest Version: https://pypi.org/project/Radicale/

How can I upgrade to actual version or do I have to wait till it's available wit pip?

Greetings

**Comment by pbiering** [comment]
this is related to https://github.com/Kozea/Radicale/issues/1483

**Comment by pbiering** [comment]
Fixed now
